src/XMakeBuildEngine/UnitTests/Evaluation/ImportFromMSBuildExtensionsPath_Tests.cs

```diff
@@ -167,6 +168,47 @@ public void ImportFromExtensionsPathWithWildCard()
         }
 
         [Fact]
+        public void ImportFromExtensionsPathWithWildCardAndSelfImport()
```

Comment:
I'm not completely sure I follow. But I did update it, is this what you mean? One of the projects that is imported has an import in it? This doesn't cover any additional code/scenario from my last one so I'm not completely sure this is what you mean. Let me know if it's not. It does make the coverage a little more obvious though, so if that's all your suggesting then how's this?